### PR TITLE
Update activitystreams for sails v0.10.5

### PR DESCRIPTION
**Pull request by dughetti**
@lsvx, @maxicecilia Hey guys, Take a look on this please!. I updated activitystreams with the new configurations that sails 0.10.5 requires.

### CODE REVIEW

File: api/responses/badRequest.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (lsvx) [inline api/responses/badRequest.js]: These are just copied straight from a new Sails project, right?

File: api/services/Caching.js [hunk 2]
```diff
@@ -90,16 +90,12 @@ module.exports = {
...
+        var cacheHash = {
                 data: data,
                 /**
-                 * Generate a proper ETag for the data. This method is particular
-                 * to the version of Express (used by Sails) used. Modern experss
-                 * versions use crypto.createHash('md5').update(JSON.stringify(data)).digest('base64')
+                 * Generate a proper ETag for the data.
                  */
-                etag: '"' + crc32.signed(JSON.stringify(data, replacer, spaces)) + '"'
+                etag: '"' + crypto.createHash('md5').update(JSON.stringify(data)).digest('base64') + '"'
```
Comment (lsvx) [inline api/services/Caching.js]: Thank god. The way etags were computed in Express 3 was so silly.

**Comment by lsvx** [comment]
Great job, @dughetti! Let's get this merged and onto staging!

File: api/responses/badRequest.js (hunk 1 above)
Comment (dughetti) [inline api/responses/badRequest.js]: Yeah, the "responses" folder is new on Sails project. We had `config/400.js`, `config/404.js`, `config/500.js` and `config/403.js` before.

File: api/services/Caching.js (hunk 2 above)
Comment (dughetti) [inline api/services/Caching.js]: :+1:

**Pull request merged**